include/sproutconnection.h

```diff
@@ -24,12 +24,17 @@ class SproutConnection
                                        const std::vector<std::string>& default_public_ids,
                                        const std::vector<std::string>& impis,
                                        SAS::TrailId trail);
+  virtual HTTPCode change_associated_identities(const std::string& default_id,
+						const std::string& user_data_xml,
```

Comment:
Alignment - I'm not going to mention it again, but you've got quite alot of misaligned lines - can you go through and check this?